config/core/test-configs.yaml

```diff
@@ -1342,6 +1342,7 @@ device_types:
     context:
       arch: x86_64
       cpu: 'qemu64'
+      memory: 1024
```

Comment:
Why was this needed?

Reply:
The default 512M seems too small. When running ltp/kselftest+ qemu related cases, there will be random OOM crash problems. Especially in ltp-mm, kselftest-lib ...

Reply:
OK thanks, let's see if we can have ltp-timers working with the default memory size on QEMU and then if it's needed for other tests we can potentially increase it.

One issue is that some labs may be configured for a certain maximum amount of memory per QEMU instance, so changing the memory attribute may not be enough if there isn't enough actual physical RAM on the machines where the jobs are run.  We might need to add an attribute in the test plan definitions about the amount of memory required so labs can enable only those that they will be able to run.  But that would be a discussion for a follow-up PR I think.